theme/ckeditor5-image/imageuploadicon.css

```diff
@@ -7,7 +7,7 @@
 	--ck-color-image-upload-icon: hsl(0, 0%, 100%);
 	--ck-color-image-upload-icon-background: hsl(120, 100%, 27%);
 
-	--ck-image-upload-icon-size: 1.25em;
+	--ck-image-upload-icon-size: 20px;
```

Comment:
Why is this a variable?

Reply:
And the below too.

Reply:
cc @oleq 

Do we want to allow modifying even such styles via CSS vars?

Reply:
`/* To make animation scalable. */` 😄 If someone decide that icon upload is too small then just one CSS Variable will change it and "tick" animation will be okey too. But we can get rid of this.

Reply:
Can't we make it bigger so people don't have to scale it? :D

Reply:
Jokes aside – it's a different story to allow easily adjusting the look of CKE5 to your app, so it fits well. And it's a different thing to allow tweaking the size of a self-contained thing like this icon.

Reply:
TBH, I'm OK with it :P We use vars for similar purposes somewhere already so it's not a big deal.